test/api/testclient/client.go

```diff
@@ -0,0 +1,41 @@
+package testclient
+
+import (
+	"os"
+
+	"github.com/konveyor/tackle2-hub/addon"
+	"github.com/konveyor/tackle2-hub/api"
+)
+
+type RestClient struct {
+	client *addon.Client
+}
+
+//var restClient, _ = NewHubClient()
+
+// Add with and without login/token creation
+func NewHubClient() (client *addon.Client, err error) {
+	client = addon.NewClient(os.Getenv("HUB_ENDPOINT"), "")
```

Comment:
Let's consider using `HUB_BASE_URL` already defined in 'settings'.

Reply:
Also, why not just do all of this in package `init()` instead of a new client object and functions?
See what I suggested on the issue.

Reply:
Yes, will update it (the ```init()```method as suggested fits great for this purpose, I'm updating the (Test)Client package).